src/api/flash.js

```diff
@@ -176,6 +180,10 @@ export const flash = async (flasher, board, port, filename, options) => {
       await delay(2000);
```

Comment:
Would it make sense to just move this "delay 2000" to the beginning of the while loop instead of adding another delay 2000 before the loop and again after each iteration?

Reply:
That makes sense